maven-release-manager/src/main/java/org/apache/maven/shared/release/config/ReleaseDescriptorBuilder.java

```diff
@@ -401,6 +401,18 @@ public ReleaseDescriptorBuilder setProjectVersionPolicyId( String projectVersion
         return this;
     }
 
+    /**
+     * <p>setProjectVersionPolicyConfig.</p>
+     *
+     * @param setProjectVersionPolicyConfig a {@link java.lang.String} object
+     * @return a {@link org.apache.maven.shared.release.config.ReleaseDescriptorBuilder} object
+     */
+    public ReleaseDescriptorBuilder setProjectVersionPolicyConfig( String setProjectVersionPolicyConfig )
+    {
+        releaseDescriptor.setProjectVersionPolicyConfig( setProjectVersionPolicyConfig );
```

Comment:
So if this would be the `XmlPlexusConfiguration` you'd be force to parse the serialized config into that type?

Reply:
No, it is an XML. But other than that you can parse it any way you like.
See for example my current implementation that uses Jackson.
https://github.com/nielsbasjes/conventional-commits-maven-release/blob/main/src/main/java/nl/basjes/maven/release/version/conventionalcommits/ConventionalCommitsVersionConfig.java

Reply:
I see, but your `toString()` does not spit out XML: https://github.com/nielsbasjes/conventional-commits-maven-release/blob/56d1e01481150ebb2e3f5afea111effcdfa25b74/src/main/java/nl/basjes/maven/release/version/conventionalcommits/ConventionalCommitsVersionConfig.java#L122-L129 since you use it at a higher layer, right?